src/components/forms/ProxyDeviceForm.tsx

```diff
@@ -108,8 +108,10 @@ const ProxyDeviceForm: FC<Props> = ({ formik, project }) => {
           options={options}
           help={<ConfigFieldDescription description={help} />}
           className="u-no-margin--bottom"
-          disabled={disabledText != undefined}
-          title={disabledText}
+          disabled={
+            disabledText != undefined || !!formik.values.editRestriction
```

Comment:
```suggestion
            !!disabledText || !!formik.values.editRestriction
```
I think we should make this more consistent.